cmd/tailscale/cli/status.go

```diff
@@ -89,6 +84,15 @@ func runStatus(ctx context.Context, args []string) error {
 			return nil, ctx.Err()
 		}
 	}
+}
+
+func runStatus(ctx context.Context, args []string) error {
+	c, bc, ctx, cancel := connect(ctx)
+	defer cancel()
+
+	bc.AllowVersionSkew = true
+
+	getStatus := getStatus(ctx, c, bc)
```

Comment:
shadowing getStatus with a new type is a little gross.

what was wrong with the getStatus closure before?

Reply:
I needed to use the getStatus closure somewhere else, so I lifted it into a function to generate the closure. I renamed the closure generator to avoid shadowing this name.